src/WPGraphQL.php

```diff
@@ -422,50 +424,112 @@ public function init_admin() {
 	 * @return void
 	 */
 	public static function show_in_graphql() {
+		add_filter( 'register_post_type_args', [ __CLASS__, 'setup_default_post_types' ], 10, 2 );
+		add_filter( 'register_taxonomy_args', [ __CLASS__, 'setup_default_taxonomies' ], 10, 2 );
 
-		global $wp_post_types, $wp_taxonomies;
+		// Run late so the user can filter the args themselves.
+		add_filter( 'register_post_type_args', [ __CLASS__, 'set_wp_type_args' ], 99, 2 );
+		add_filter( 'register_taxonomy_args', [ __CLASS__, 'set_wp_type_args' ], 99, 2 );
+	}
 
+	/**
+	 * Sets up the default post types to show_in_graphql.
+	 *
+	 * @param array  $args      Array of arguments for registering a post type.
+	 * @param string $post_type Post type key.
+	 *
+	 * @return array
+	 */
+	public static function setup_default_post_types( $args, $post_type ) {
```

Comment:
@jasonbahl I wanted to move this and `setup_default_taxonomies()` into `Registry\Utils\{Post|Term}Object`, but not sure if that still makes sense given the `Utils` namespace.  

Thoughts?